docs/set_up_transfer_and_export_device.rst

```diff
@@ -0,0 +1,210 @@
+.. |screenshot of passphrase selection prompt in the Disk Utility application| image:: images/screenshots/create-passphrase.png
+
+.. [#] Tails screenshots were taken on Tails 3.0.1. Please make an issue on
+       GitHub if you are using the most recent version of Tails and the
+       interface is different from what you see here.
+
+Create a USB *Export Device*
+----------------------------
+We recommend using a fully encrypted USB drive for coping files off the
+*Secure Viewing Station*. This is even more important than for the
+*Transfer Device*, as the risk of accidentally leaving decrypted files on the
+*Export Device* is significant.
+
+Because the *Export Device* will need to be mounted on both Tails and the
+journalist's everyday workstation, you will need to use an encryption scheme
+that works on both operating systems.
+
+This is the case for many hardware-encrypted USB drives. When considering
+a hardware solution, we recommend selecting a vendor that has fully opened
+the source code and specifications of their devices and encouraged third party
+audits.
+
+VeraCrypt is a good alternative to hardware-based encryption. It is actively
+maintained cross-platform software that has been independently audited and is
+free to use.
+
+VeraCrypt-encrypted media can be opened in the Tails operating system without
+installing additional software. However, to *create* your encrypted VeraCrypt
+drive in the first place, you will need to install the VeraCrypt software. The
+`guide by Freedom of the Press Foundation <https://freedom.press/training/encryption-toolkit-media-makers-veracrypt-guide/>`__
+provides instructions for encrypting storage media using VeraCrypt.
+
+As with the *Transfer Device*, we recommend storing the passphrase in the
+password manager of each user who will use a given *Export Device*.
+
+Limiting write access
+~~~~~~~~~~~~~~~~~~~~~
+If you re-use the same *Export Device* for multiple copy operations, there is
+the risk of introducing malware to the *Secure Viewing Station* from your
+network. Depending on your threat model, there are steps you may want to take to
+mitigate that risk.
+
+One option is to restrict write access to the *Export Device* before it is
+plugged into a device other than the *Secure Viewing Station*. `Write blockers <https://www.forensicswiki.org/wiki/Write_Blockers>`__
+are used in forensics to ensure storage media are not modified during
+examination. At a lower cost, some USB flash drives come with a physical write
+protection switch, but it may be more difficult to consistently remember or
+enforce its usage.
+
+Full-size SD cards also come with physical write protection switches. However,
+this write protection is fully host-based (the host operating system can choose
+to ignore it), and should therefore be considered less secure against
+sophisticated malware.
+
+Please see our :doc:`journalist guide <journalist>` for additional
+recommendations regarding malware mitigation.
```

Comment:
As above, a link to the "Working with Documents" section within the Journalist Guide would be great.

Reply:
Done!